src/Playwright.TestAdapter/PlaywrightSettingsProvider.cs

```diff
@@ -40,7 +40,8 @@ public static string BrowserName
         get
         {
             var browserFromEnv = Environment.GetEnvironmentVariable("BROWSER")?.ToLowerInvariant();
-            if (!string.IsNullOrEmpty(browserFromEnv))
+            // GitHub Codespaces sets the BROWSER environment variable, ignore it if its bogus.
+            if (!string.IsNullOrEmpty(browserFromEnv) && !browserFromEnv!.StartsWith("/vscode/"))
```

Comment:
Ouch! I don't like hardcoding this.

Perhaps we should change the error below into a warning and run `chromium`, if we consider `BROWSER` to be a legacy thing?